cypress/integration/spec_multiuser_sessions.js

```diff
@@ -0,0 +1,92 @@
+describe('Test the actions in UTXO list', () => {
+    // See https://github.com/cypress-io/cypress/issues/21138
+    if (Cypress.env("CI")) {
+        const login = (name,password) => {
```

Comment:
Perhaps I am overlooking sth. but where is the name variable set?

Reply:
It's defining a function and the name is the parameter here. It's initialized with the arguments at call time, e.g. in line 39.